### ISSUE TITLE
Show dependent nodes from context menu

### ISSUE THREAD

**Issue by alschmut** [MaibornWolff/codecharta#218]
Original feature request was #172. Depends on #211.

Use [feature/172/visualizing-edges](https://github.com/MaibornWolff/codecharta/tree/feature/172/visualizing-edges) to checkout a branch.

## Description
By right-clicking a building the context menu appears and should give an added option to "show dependent nodes":
- This would make all NOT-dependent nodes flat, maybe even in a grey/white tone
- The dependent nodes stay as they are and use the normal width-key, height-key and color-key from the metricChooser (see #217 for using the edge-attributes)
- The arrows will be shown as well, but can be removed by a checkbox inside the settings-sidebar

<img width="70%" src="https://user-images.githubusercontent.com/23529226/44145321-6a9e179c-a08b-11e8-9686-03d5dbcb363c.jpg">

**Comment by alschmut** [comment]
solved and merged into [feature/172/visualizing-edges](https://github.com/MaibornWolff/codecharta/tree/feature/172/visualizing-edges)
